Raptacon/Robot-2023
# Arm extension

## Types of changes


## Associated Issue(s)
#25 

## Steps to Test
Use the right stick to control the arm extension. It should change the current position in shuffleboard.

## Changelog
With an extending arm we need ways to move it especially with an analog stick as requested by drive team.

## Why?
I am making this change so we can use our extending arm.
